src/components/manage/Blocks/Teaser/utils.js

```diff
@@ -1,14 +1,14 @@
 import { isInternalURL } from '@plone/volto/helpers';
 import config from '@plone/volto/registry';
 
-export function getTeaserImageURL({ href, image, align }) {
+export function getTeaserImageURL({ href, image, align, scale }) {
```

Comment:
To keep backwards compatibility in the function, I think that the scale parameter should get a default value.

```javascript
export function getTeaserImageURL({ href, image, align, scale = 'teaser' }) {
```